MANIFEST.in

```diff
@@ -0,0 +1,17 @@
+graft django_redshift_backend
+
+include README.rst
+include tox.ini
+
+exclude checklist.rst
+exclude Dockerfile
+
+global-exclude __pycache__
+global-exclude *.py[co]
+global-exclude *.cfg
+global-exclude *.ini
+global-exclude *.txt
+global-exclude *.yaml
+
+prune .github
+prune tests
```

Comment:
I've never seen `check-manifest` tool before, but if this content is the result of using the `check-manifest` tool, I think it's too much.